server/service/src/sync/test/test_data/purchase_order.rs

```diff
@@ -0,0 +1,814 @@
+use crate::sync::{
+    test::TestSyncIncomingRecord,
+    translations::purchase_order::{
+        LegacyPurchaseOrderRow, LegacyPurchaseOrderStatus, PurchaseOrderOmsFields,
+    },
+};
+use chrono::NaiveDate;
+use repository::{PurchaseOrderRow, PurchaseOrderStatus};
+use serde_json::json;
+
+use super::TestSyncOutgoingRecord;
+
+const TABLE_NAME: &str = "purchase_order";
+
+// populated object case
+const PURCHASE_ORDER_1: (&str, &str) = (
+    "12e889c0f0d211eb8dddb54df6d741hx",
+    r#"{
+        "name_ID": "name_store_b",
+        "ID": "12e889c0f0d211eb8dddb54df6d741hx",
+        "creation_date": "2021-01-22",
+        "target_months": 2.1,
+        "status": "nw",
+        "serial_number": 1,
+        "store_ID": "store_b",
+        "comment": "some test comment",
+        "currency_ID": "currency_a",
+        "inv_sub_total": 12.1,
+        "freight": 0.24,
+        "cost_in_local_currency": 12.34,
+        "curr_rate": "",
+        "reference": "test reference",
+        "lines": "",
+        "requested_delivery_date": "",
+        "locked": "",
+        "confirm_date": "2021-07-11",
+        "created_by": "some user",
+        "last_edited_by": "some other user",
+        "Order_total_after_discount": 12.2,
+        "supplier_agent": "some agent",
+        "authorizing_officer_1": "agent",
+        "authorizing_officer_2": "agent2",
+        "freight_conditions": "difficult",
+        "additional_instructions": "additional instructions",
+        "agent_commission": 1.0,
+        "document_charge": 0.5,
+        "communications_charge": 0.0,
+        "freight_charge": 0,
+        "total_foreign_currency_expected": "",
+        "total_local_currency_expected": "",
+        "insurance_charge": 1.0,
+        "Order_total_before_discount": "",
```

Comment:
``` suggestion
        "Order_total_before_discount": 0.0,
```

Will also fix tests. And I am assuming safe given that og float types won't ever be an empty string?